integration_tests/tests/fixtures/fixture.rs

```diff
@@ -14,7 +14,8 @@ use solana_sdk::{
 use spl_associated_token_account::{
     get_associated_token_address, instruction::create_associated_token_account_idempotent,
 };
-use spl_token::state::{Account, Mint};
+use spl_token::state::Mint;
```

Comment:
Generally, you should use the `spl_token_2022` library for everything since it is backwards compatible

Reply:
If you want some extra context, I have a PR open for Token2022 that we're delaying on a bit. It might be useful to look through:

https://github.com/jito-foundation/restaking/pull/98